rpc/src/main/protobuf/master.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package kr.ac.postech.paranode.rpc;
+
+service Master {
+    
+    rpc RegisterWorkerDirectory (RegisterRequest) returns (RegisterRelpy) {}
+}
+
+message RegisterRequest {
+    string ipAddress = 1;
+    repeated string inputDirectory = 2;
+    string outputDirectory = 3;
+}
+
+message RegisterRelpy {
```

Comment:
its also changed to `RegisterReply`

Reply:
# https://github.com/betarixm/434project/pull/59/commits/f7b490679dd9d46949c4e52e45f3946ccaf3f8c1